config.toml

```
@@ -356,6 +356,10 @@ logo = "pro-logo.svg"
 url = "https://www.purefacts.com"
 logo = "purefacts.png"
 
+[[params.qonto]]
```

This will need to be aligned
```suggestion
[[params.users]]
```

---

Sorry, too tired. I fixed and rebased. Thank you.